### PR TITLE
Output the output

### PR DESCRIPTION
**Pull request by asiviero**
💅 by outputting the output, so we can create PRs straight from that

bless gitgud

### CODE REVIEW

File: bin/gitgud [hunk 1]
```diff
@@ -3,23 +3,25 @@
...
-    elsif !output.match?(/remote: Internal Server Error\sEverything up-to-date/)
-      spinner.update_title("Encountered error while git pushing\n#{output}")
+    elsif !$output.match?(/remote: Internal Server Error\sEverything up-to-date/)
+      spinner.update_title("Encountered error while git pushing\n#{$output}")
       return :task_failed
     end
   end
 
-  spinner.update_title("Unsuccessful after 20 attemps")
+  spinner.update_title("Unsuccessful after 20 attempts")
```
Comment (jusleg) [inline bin/gitgud:24]: sheesh

File: bin/gitgud [hunk 2]
```diff
@@ -45,7 +47,10 @@ end
 def run_da_whole_show
   if ARGV[0] == 'push'
     ARGV.shift
+    CLI::UI::Frame.open('Gittin gud')
```
Comment (jusleg) [inline bin/gitgud:50]: VERY NICE 🔥

**Review by jusleg — APPROVED** [review]
giphy

File: bin/gitgud (hunk 2 above)
Comment (asiviero) [reply bin/gitgud:50]: image

**Pull request merged**